### PR TITLE
0.2.0 release

### PR DESCRIPTION
**Pull request by LukeMathWalker**
Getting ready for releasing - I have added a new entry to the changelog, updated dependencies where needed and put some more info on the README.md.

Are we ready to go? What do you think @jturner314?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,56 @@
...
+      * Cross-entropy
+      * Kullback-Leibler divergence
+    * Quantiles and order statistics:
+      * `argmin` / `argmin_skipnan`
+      * `argmax` / `argmax_skipnan`
+      * Optimized bulk quantile computation (`quantiles_mut`, `quantiles_axis_mut`)
+  * Fixes:
+    * Reduced occurrences of overflow for `interpolate::midpoint`
+  * Improvements:
+    * Redesigned error handling across the whole crate (standardise on `Result` with `ErrorKind` where needed)
```
Comment (jturner314) [inline README.md]: What do you mean by "with `ErrorKind` where needed"?

Comment (LukeMathWalker) [reply README.md]: I meant the enumeration of possible failure types. Given that this is a high-level overview, I should probably drop that detail.

**Comment by LukeMathWalker** [comment]
I merged all of them - things we already discussed and agreed upon.
I have added a note on extension traits to the changelog, as you suggested.

**Comment by LukeMathWalker** [comment]
Let me try, let's see if I can get it right :stuck_out_tongue:

**Comment by jturner314** [comment]
@LukeMathWalker Thanks for doing the release. Will you please add a `0.2.0` tag to the commit in Git and push the tag to GitHub?

**Comment by LukeMathWalker** [comment]
Done! (although directly from GitHub GUI)

**Pull request merged**